src/components/Footer/index.tsx

```diff
@@ -12,15 +12,31 @@ const Footer = () => {
         <p className={styles.FooterBodyContent}>
           Discuss ideas and share solutions with developers worldwide.
         </p>
-        <a
-          data-testid='community-link'
-          style={{ textDecoration: 'none' }}
-          href='https://binary.vanillacommunity.com/'
-          target='_blank'
-          rel='noreferrer'
-        >
-          <button className={styles.communityButton}>Join our community</button>
-        </a>
+        <div className={styles.Communities}>
+          <a
+            data-testid='community-link'
+            style={{ textDecoration: 'none' }}
+            href='https://deriv.vanillacommunity.com/'
+            target='_blank'
+            rel='noreferrer'
+          >
+            <button className={styles.communityButton}>Join our community</button>
```

Comment:
do we have to add `button` element inside `a` tag? 
can we use other tags here? 
and for the next one as well.

Reply:
Okay. Thanks!